i18n/create_messages.py

```diff
@@ -34,15 +34,15 @@ def string_is_ascii(s):
     return True
   except UnicodeEncodeError:
     return False
-  
+
 def load_constants(filename):
   """Read in constants file, which must be output in every language."""
   constant_defs = read_json_file(filename);
   constants_text = '\n'
   for key in constant_defs:
     value = constant_defs[key]
     value = value.replace('"', '\\"')
-    constants_text += '\nBlockly.Msg.{0} = \"{1}\";'.format(key, value)  
```

Comment:
Taking another look, there are quite a few places in code where we're using Blockly.Msg.SOMETHING, so the advanced optimizations might still be broken. @export might be an option instead of updating all of the other refs and then we can change the [parser in utils](https://github.com/google/blockly/blob/master/core/utils.js#L554) to use the exported style.